src/helpers/isSandboxLinked.js

```diff
@@ -14,8 +14,10 @@ const fs = require('fs');
 
 module.exports = () => {
   try {
-    const stat = fs.lstatSync('node_modules/@adobe/reactor-sandbox');
-    return stat.isSymbolicLink();
+    // We no longer test for a symbolic link because of pnpm.
+    // Probably the safest way to detect if we are doing developer
+    // work on sandbox is to search for the Git directory.
+    return fs.existsSync('node_modules/@adobe/reactor-sandbox/.git');
```

Comment:
Tested against `npm` and `pnpm`?

Reply:
Yes

Reply:
Sounds like this change doesn't break using `npm`?

Reply:
NPM should work like it did until now.